apps/web/src/app/(dashboard)/(management)/locatie/[location]/cohorten/[cohort]/(overview)/diplomas/_components/table-actions.tsx

```diff
@@ -218,6 +220,7 @@ export function IssueCertificateDialog({
               cohortId,
             })
               .then(() => setIsOpen(false))
+              .then(() => resetSelection())
```

Comment:
Is there a reason to put this in a second `then` call instead of batching it in one?